packages/storefront-lwc/scripts/runtime.mjs

```diff
@@ -27,6 +27,28 @@ const publicDir = `${templateDir}/dist/public/`;
 const port = process.env.PORT || 3002;
 const mode = process.env.NODE_ENV || 'development';
 
+function validateConfig(config) {
+    const REQUIRED_KEYS = [
+        'COMMERCE_API_PATH',
+        'COMMERCE_CLIENT_API_SITE_ID',
+        'COMMERCE_CLIENT_CLIENT_ID',
+        'COMMERCE_CLIENT_REALM_ID',
+        'COMMERCE_CLIENT_INSTANCE_ID',
+        'COMMERCE_CLIENT_ORGANIZATION_ID',
+        'COMMERCE_CLIENT_SHORT_CODE',
```

Comment:
I added some validation of the config file / env variables. If one of these is not present, the app will tell you its needed and exit.